src/main/target/common_pre.h

```diff
@@ -62,6 +62,7 @@
 #define USE_MOTOR
 #define USE_DMA
 #define USE_TIMER
+#define USE_RPM_LIMITER
```

Comment:
We are going to add this as custom define. So no need to add this here as we should add to cloud build options or custom defines and update documentation